src/index/ExternalVocabulary.cpp

```diff
@@ -3,6 +3,9 @@
 // Author: Björn Buchhold <buchholb>
 
 #include "./ExternalVocabulary.h"
+
+#include <fstream>
```

Comment:
We use our own `util::File` class everywhere else I mean it clearly has it's faults but we should probably keep things consistent.

Reply:
I would argue that for this cas (reading and writing text files line by line), std::getline on an fstream is better style and, according to a comment in the util::File class, also more performant. I would argue towards refactoring towards the std library when using textfiles in a line-by-line manner and util::File when reading and writing binary, because there some wrapping can not be wrong.

Reply:
Interesting idea, doing a quick search it looks like the only (non test) use of `File::readLine()` is in the [Text Index](https://github.com/ad-freiburg/QLever/blob/3667d45cc625610106ce82465ffbda5600745071/src/index/Index.Text.cpp#L42) reading docs files. So yeah I think it wouldn't be too much work getting rid of this use of `File`.